capa/features/extractors/helpers.py

```diff
@@ -52,6 +52,9 @@ def generate_symbols(dll: str, symbol: str) -> Iterator[str]:
       - CreateFileA
       - CreateFile
     """
+    # normalize dll name
+    dll = dll.lower()
+
```

Comment:
I believe this is the best place to normalize symbol names as all of the extractors use this function to emit `Import` and `API` features.